examples/redux-saga/dynamic-counters/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Counter from './components/Counter'
+import { ADD_COUNTER } from './actionTypes'
+
+const initialState = { counterIds: ['Counter one', 'Counter two'] }
+
+export const pageReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_COUNTER:
+      return { ...state, counterIds: [...state.counterIds, action.value] }
+    default:
+      return state
+  }
+}
+
+class DynaCounter extends React.Component {
```

Comment:
Move to the `compnents` folder.  Could add a link to it directly in the README.

Reply:
moved and linked